nginx.conf

```diff
@@ -20,20 +20,29 @@
 		ssl_trusted_certificate	/srv/nginx/ca-certs.pem;
 		ssl_dhparam		/srv/nginx/dhparam.pem;
 
-		# Authentification
-		set			$setup_type ovf-ami;
-		set			$realm off;
+        # Authentification
+        set $setup_type ovf-ami;
+        set $realm off;
```

Comment:
Can we revert those tabs/spaces changes so it is easier to see what was changed? Because right now it looks like all changes are with tabs/spaces, and nothing really changed.

Reply:
I've tried to replace spaces with tabs, but Github didn't get these changes, so I've decided to create a diff between old and new **nginx.conf**:
```diff
--- nginx.conf.old      2020-07-03 10:47:42.424314393 +0300
+++ nginx.conf  2020-07-03 10:52:28.293882854 +0300
@@ -24,15 +24,24 @@
                set                     $setup_type ovf-ami;
                set                     $realm off;
 
+               if (-f /.pmm-ovf) {
+                       set                     $setup_type ovf;
+               }
+
+               if (-f /.pmm-ami) {
+                       set                     $setup_type ami;
+               }
+
                if (-f /srv/nginx/.htpasswd) {
-                       set             $realm on;
-                       set             $setup_type "${setup_type}-protected";
+                       set                     $realm on;
                }
-               if ($setup_type = "ovf-ami") {
-                       rewrite         ^/$ $scheme://$http_host/password-page/ permanent;
+
+               if ($setup_type ~ "^(ovf|ami)$") {
+                       rewrite ^/$ $scheme://$http_host/password-page/$setup_type/     permanent;
                }
-               if ($setup_type ~ "-protected") {
-                       rewrite         ^/password-page $scheme://$http_host/ permanent;
+
+               if ($realm = "on") {
+                       rewrite ^/password-page/$setup_type/    $scheme://$http_host/   permanent;
                }
 
                root                    /usr/share/pmm-server/landing-page;
```